### PR TITLE
feat: balsamic report tags

### PR DESCRIPTION
**Pull request by ivadym**
## Description
This PR adds new BALSAMIC report tags:

---

WGS cases:

`QC and reports` and `Raw data files and Aligned files` remain the same as the panel cases.

**Variant-called vcf files**

#### Somatic SNV

- [x] .tnscope.all.vcf.gz, (HK)
- [x] .tnscope.all.vcf.gz.tbi, (HK)
- [x] .tnscope.all.filtered.pass.vcf.gz, (HK)
- [x] .tnscope.all.filtered.pass.vcf.gz.tbi, (HK)
- [x] .tnhaplotyper.all.vcf.gz, (HK)
- [x] .tnhaplotyper.all.vcf.gz.tbi, (HK)
- [x] .tnhaplotyper.all.filtered.vcf.gz --> _not mandatory_
- [x] .tnhaplotyper.all.filtered.vcf.gz.tbi --> _not mandatory_
- [x] .tnhaplotyper.all.filtered.pass.vcf.gz, (HK)
- [x] .tnhaplotyper.all.filtered.pass.vcf.gz.tbi, (HK)

### Somatic SV

- [x] .manta.all.vcf.gz, (HK) --> _already defined 
- [x] .manta.all.vcf.gz.tbi, (HK) --> _already defined 
- [x] .manta.all.filtered.pass.vcf.gz, (HK, Scout) --> _already defined 
- [x] .manta.all.filtered.pass.vcf.gz.tbi, (HK, Scout) --> _already defined
- [x] .delly.all.vcf.gz
- [x] .delly.all.vcf.gz.tbi
- [x] .delly.all.filtered.pass.vcf.gz, (HK) --> _not mandatory_
- [x] .delly.all.filtered.pass.vcf.gz.tbi, (HK) --> _not mandatory_
- [ ] .delly.bcf, (HK) --> No housekeeper ID in Balsamic
- [ ] .delly.bcf.csi (HK) --> No housekeeper ID in Balsamic

### Somatic CNVs

- [x] .ascat.all.vcf.gz, (HK)
- [x] .ascat.all.vcf.gz.tbi, (HK)
- [x] .ascat.all.filtered.pass.vcf.gz, (HK)
- [x] .ascat.all.filtered.pass.vcf.gz.tbi, (HK)
  
 ### Germline SV

- [x] .normal.manta_germline.vcf.gz, (HK) --> _already defined (not mandatory for WGS)_
- [x] .normal.manta_germline.vcf.gz.tbi, (HK) --> _already defined (not mandatory for WGS)_
- [x] .tumor.manta_germline.vcf.gz, (HK, Scout) --> _already defined (not mandatory for WGS)_
- [x] .tumor.manta_germline.vcf.gz.tbi,  (HK, Scout) --> _already defined (not mandatory for WGS)_

### Germline SNV

- [x] .normal.dnascope.vcf.gz, (HK) --> _already defined (not mandatory for WGS)_
- [x] .normal.dnascope.vcf.gz.tbi, (HK) --> _already defined (not mandatory for WGS)_
- [x] .tumor.dnascope.vcf.gz, (HK) --> _already defined (not mandatory for WGS)_
- [x] .tumor.dnascope.vcf.gz.tbi, (HK) --> _already defined (not mandatory for WGS)_

also  include:

- [ ] *.all.stats
- [ ] *_summary.html

## Additional new WGS plots to include

- [x] /home/proj/stage/cancer/cases/$wgs_caseid/analysis/vcf/*.ascat.ascatprofile.png (HK)
- [x] /home/proj/stage/cancer/cases/$wgs_caseid/analysis/vcf/*.ascat.ASPCF.png (HK)
- [x] /home/proj/stage/cancer/cases/$wgs_caseid/analysis/vcf/*.ascat.germline.png (HK)
- [x] /home/proj/stage/cancer/cases/$wgs_caseid/analysis/vcf/*.ascat.rawprofile.png (HK)
- [x] /home/proj/stage/cancer/cases/$wgs_caseid/analysis/vcf/*.ascat.sunrise.png (HK)
- [x] /home/proj/stage/cancer/cases/$wgs_caseid/analysis/vcf/*..ascat.tumor.png (HK)
- [x] /home/proj/stage/cancer/cases/$wgs_caseid/analysis/vcf/*.ascat.samplestatistics.txt (HK)

### CODE REVIEW

File: cg_hermes/config/tags.py [hunk 1]
```diff
@@ -119,6 +119,8 @@
     "chanjo": {"description": "Tool to keep track of coverage over specific regions"},
     "chromograph": {"description": "Tool to create PNG images from BED and WIG files from mikaell"},
     "cnvkit": {"description": "Tool to call copy number variations"},
+    "ascat-ngs": {"description": "Tool to identify somatically acquired copy-number alterations"},
+    "delly": {"description": "Structural variant prediction tool"},
```
Comment (ashwini06) [inline cg_hermes/config/tags.py:123]:
```suggestion
    "delly": {"description": "Cancer structural variant prediction tool "},
```

can you also remove `mutect` and `strelka` from below lines. As these callers no more exists in BALSAMIC

File: docs/all_tags.md [hunk 2]
```diff
@@ -68,6 +68,8 @@
 | chanjo           | Tool to keep track of coverage over specific regions          |
 | chromograph      | Tool to create PNG images from BED and WIG files from mikaell |
 | cnvkit           | Tool to call copy number variations                           |
+| ascat-ngs        | Tool to identify somatically acquired copy-number alterations |
+| delly            | Structural variant prediction tool                            |
```
Comment (ashwini06) [inline docs/all_tags.md:72]: Same here remove `mutect` and `strelka` from below lines

**Review by ashwini06 — APPROVED** [review]
Looks good 👍🏼  Just a general comment, if you come across old variant callers `strelka` and `mutect` can you remove them, as they are no longer exist in BALSAMIC

**Comment by ivadym** [comment]
Updated manually `fleetjay` case on `stage` with the new `ascat-ngs` tags for visualization and tested on `hermes-stage`:

Screen Shot 2021-11-24 at 12 59 43

**Pull request merged**